docs/cow-protocol/affiliate-program/faq.md

```
@@ -0,0 +1,277 @@
+---
+title: Affiliate Program FAQ
+sidebar_label: FAQ
+sidebar_position: 2
+---
+
+### Overview
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)